storage/objects/objects_test.go

```diff
@@ -479,14 +485,22 @@ func TestPostPolicyV4(t *testing.T) {
 		t.Fatalf("failed to compose HTTP request: %v", err)
 	}
 	req.Header.Set("Content-Type", mw.FormDataContentType())
+
+	// Dump the request for logging.
+	requestDump, err := httputil.DumpRequest(req, true)
+	if err != nil {
+		t.Logf("error dumping request for logging: %v", err)
+	}
+
 	res, err := http.DefaultClient.Do(req)
 	if err != nil {
 		t.Fatalf("client.Do: %v", err)
 	}
 	if g, w := res.StatusCode, 204; g != w {
 		blob, _ := httputil.DumpResponse(res, true)
```

Comment:
Let's call this responseDump

Reply:
Shouldn't it be `requestDump`?

Reply:
No, `res` is the HTTP response. We want to make sure to log BOTH the request we are sending and the response from the server.

Reply:
I don't quite understand - the lines here are dumping the request body into a variable before we send it so we can log it later if needed. The response gets dumped later on (line 500) if there is an unexpected result.